Makefile

```
@@ -13,6 +13,8 @@ PAPEROPT_letter = -D latex_paper_size=letter
 ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
+# Explicitly set SOURCE_DATE_EPOCH for Sphinx copyright
+SOURCE_DATE_EPOCH = $(shell date +%s)
```

```suggestion
SOURCE_DATE_EPOCH = $(shell date +%s --date='Jan 1')
```

This will make it use the unix epoch of 1 January of the current year, so it's at least reproducible within the same year :)